src/middlewared/middlewared/main.py

```diff
@@ -377,7 +377,14 @@ def register_job(self, job_id):
         self.jobs[job_id] = self.middleware.loop.call_later(
             60, lambda: asyncio.ensure_future(self._cleanup_job(job_id)))
 
+    async def _cleanup_cancel(self, job_id):
+        job = self.jobs.pop(job_id, None)
```

Comment:
Took me a while to understand what's going on. Can we name it `job_cleanup`? For me it looked like we are canceling the job itself.

Reply:
We already have a _cleanup_job method. Why would we call it job_cleanup if its really canceling a future?

Reply:
@william-gr I mean name variable `job_cleanup`